components/dashboard/src/settings/Teams.tsx

```diff
@@ -49,29 +50,73 @@ function AllTeams() {
     const [inviteMembersModal, setInviteMembersModal] = useState<{ sub: TeamSubscription } | undefined>(undefined);
     const [addMembersModal, setAddMembersModal] = useState<{ sub: TeamSubscription } | undefined>(undefined);
 
-    const [pendingPlanPurchase, setPendingPlanPurchase] = useState<{ plan: Plan } | undefined>(getLocalStorageObject<{ plan: Plan }>("pendingPlanPurchase"));
-    const [pendingSlotsPurchase, setPendingSlotsPurchase] = useState<{ ts: TeamSubscription } | undefined>(getLocalStorageObject<{ ts: TeamSubscription }>("pendingSlotsPurchase"));
+    const restorePendingPlanPurchase = () => {
+        console.log("restorePendingPlanPurchase")
```

Comment:
leftover

Reply:
it runs 6 times after reload

Reply:
ah, indeed. will add a check for `pendingSlotsPurchase`. 

though, it's uncritical. `pendingSlotsPurchase` is updated a single time on reload.